### DISCUSSION TITLE
How to convert the MD trajectory file. xyz. ener, etc. of cp2k into input data for dpgen?

### DISCUSSION THREAD

**Discussion by chenyuping6** [deepmodeling/dpgen#1451] [Q&A]
I want to learn how to use CP2K to complete deepmd kit training, but my initial data is MD simulation completed by CP2K.How to convert the MD trajectory file. xyz. ener, etc. of cp2k into input data for dpgen? Or what documents should I provide at this time?

**Comment by moyuxuan12** [comment]
Have you solved your problem?
